Lein Impossible to find sass file on Windows

**Describe the bug (REQUIRED)**
When executing the `lein init` command on Windows 10, it fails to use the sass.bat file instead of the sass file for linux


**To Reproduce (REQUIRED)**
Simply follow the build instructions on a Windows system and an error occurs while building with lein
when following [this guide ](https://github.com/defold/defold/blob/dev/editor/README_BUILD.md)

**Expected behavior (REQUIRED)**
On windows system lein should be able to use the sass.bat file

**Defold version (REQUIRED):**
  - Version 1.6.2

**Platforms (REQUIRED):**
 - Platforms: Windows
 - OS: [Windows 
 - Device: PC

**Logs (OPTIONAL):**
```
Compiling 38 source files to C:\Defold\defold\editor\target\classes
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
using cached file 'https://github.com/sass/dart-sass/releases/download/1.62.1/dart-sass-1.62.1-windows-x64.zip' -> 'C:\Users\Burrish\.dcache\github_com-sass-dart-sass-releases-download-1.62.1-dart-sass-1.62.1-windows-x64.zip-22307838444234354535353839304435463822'
sass: styling/stylesheets/ -> resources/
java.io.IOException: Cannot run program "C:\Defold\defold\editor\target\packages\dart-sass\1.62.1\dart-sass\sass": CreateProcess error=2, The system cannot find the file specified
 at java.lang.ProcessBuilder.start (ProcessBuilder.java:1143)
    java.lang.ProcessBuilder.start (ProcessBuilder.java:1073)
    leiningen.sass$sass.invokeStatic (sass.clj:85)
    leiningen.sass$sass.doInvoke (sass.clj:71)
    clojure.lang.RestFn.applyTo (RestFn.java:139)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$apply.invoke (core.clj:662)
    leiningen.core.main$partial_task$fn__7380.doInvoke (main.clj:284)
    clojure.lang.RestFn.applyTo (RestFn.java:139)
    clojure.lang.AFunction$1.doInvoke (AFunction.java:31)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$apply.invoke (core.clj:662)
    leiningen.core.main$apply_task.invokeStatic (main.clj:334)
    leiningen.core.main$apply_task.invoke (main.clj:320)
    leiningen.core.main$resolve_and_apply.invokeStatic (main.clj:343)
    leiningen.core.main$resolve_and_apply.invoke (main.clj:336)
    leiningen.init$init.invokeStatic (init.clj:88)
    leiningen.init$init.doInvoke (init.clj:75)
    clojure.lang.RestFn.applyTo (RestFn.java:139)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$apply.invoke (core.clj:662)
    leiningen.core.main$partial_task$fn__7380.doInvoke (main.clj:284)
    clojure.lang.RestFn.applyTo (RestFn.java:139)
    clojure.lang.AFunction$1.doInvoke (AFunction.java:31)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$apply.invoke (core.clj:662)
    leiningen.core.main$apply_task.invokeStatic (main.clj:334)
    leiningen.core.main$apply_task.invoke (main.clj:320)
    leiningen.core.main$resolve_and_apply.invokeStatic (main.clj:343)
    leiningen.core.main$resolve_and_apply.invoke (main.clj:336)
    leiningen.core.main$_main$fn__7469.invoke (main.clj:453)
    leiningen.core.main$_main.invokeStatic (main.clj:442)
    leiningen.core.main$_main.doInvoke (main.clj:439)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.core$apply.invokeStatic (core.clj:667)
    clojure.main$main_opt.invokeStatic (main.clj:514)
    clojure.main$main_opt.invoke (main.clj:510)
    clojure.main$main.invokeStatic (main.clj:664)
    clojure.main$main.doInvoke (main.clj:616)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.main.main (main.java:40)
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
 at java.lang.ProcessImpl.create (ProcessImpl.java:-2)
    java.lang.ProcessImpl.<init> (ProcessImpl.java:499)
    java.lang.ProcessImpl.start (ProcessImpl.java:158)
    java.lang.ProcessBuilder.start (ProcessBuilder.java:1110)
    java.lang.ProcessBuilder.start (ProcessBuilder.java:1073)
    leiningen.sass$sass.invokeStatic (sass.clj:85)
    leiningen.sass$sass.doInvoke (sass.clj:71)
    clojure.lang.RestFn.applyTo (RestFn.java:139)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$apply.invoke (core.clj:662)
    leiningen.core.main$partial_task$fn__7380.doInvoke (main.clj:284)
    clojure.lang.RestFn.applyTo (RestFn.java:139)
    clojure.lang.AFunction$1.doInvoke (AFunction.java:31)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$apply.invoke (core.clj:662)
    leiningen.core.main$apply_task.invokeStatic (main.clj:334)
    leiningen.core.main$apply_task.invoke (main.clj:320)
    leiningen.core.main$resolve_and_apply.invokeStatic (main.clj:343)
    leiningen.core.main$resolve_and_apply.invoke (main.clj:336)
    leiningen.init$init.invokeStatic (init.clj:88)
    leiningen.init$init.doInvoke (init.clj:75)
    clojure.lang.RestFn.applyTo (RestFn.java:139)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$apply.invoke (core.clj:662)
    leiningen.core.main$partial_task$fn__7380.doInvoke (main.clj:284)
    clojure.lang.RestFn.applyTo (RestFn.java:139)
    clojure.lang.AFunction$1.doInvoke (AFunction.java:31)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:669)
    clojure.core$apply.invoke (core.clj:662)
    leiningen.core.main$apply_task.invokeStatic (main.clj:334)
    leiningen.core.main$apply_task.invoke (main.clj:320)
    leiningen.core.main$resolve_and_apply.invokeStatic (main.clj:343)
    leiningen.core.main$resolve_and_apply.invoke (main.clj:336)
    leiningen.core.main$_main$fn__7469.invoke (main.clj:453)
    leiningen.core.main$_main.invokeStatic (main.clj:442)
    leiningen.core.main$_main.doInvoke (main.clj:439)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.core$apply.invokeStatic (core.clj:667)
    clojure.main$main_opt.invokeStatic (main.clj:514)
    clojure.main$main_opt.invoke (main.clj:510)
    clojure.main$main.invokeStatic (main.clj:664)
    clojure.main$main.doInvoke (main.clj:616)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.lang.Var.applyTo (Var.java:705)
    clojure.main.main (main.java:40)
```

--- a/editor/tasks/leiningen/sass.clj
+++ b/editor/tasks/leiningen/sass.clj
@@ -66,7 +66,11 @@
         extract-path (io/file packages-path "dart-sass" version)]
     (when-not (.exists extract-path)
       (extract (http-cache/download url) extract-path ext))
-    (str (io/file extract-path "dart-sass" "sass"))))
+    (str (io/file extract-path
+                  "dart-sass"
+                  (str "sass"
+                       (when (-> (System/getProperty "os.name") .toLowerCase (.contains "win"))
+                         ".bat"))))))
 
 (defn sass [project & [mode]]
   {:pre [(#{"once" "auto"} mode)]}

